src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -351,6 +351,17 @@ def get_platform_info():
     hw_info_dict['asic_type'] = version_info['asic_type']
     hw_info_dict['asic_count'] = get_num_asics()
 
+    try:
+        config_db = ConfigDBConnector()
+        config_db.connect()
+
+        metadata = config_db.get_table('DEVICE_METADATA')["localhost"]
+        switch_type = metadata.get('switch_type')
+        if switch_type:
+            hw_info_dict['chassis_type'] = switch_type
```

Comment:
The json o/p has "chassis_type": "voq". 
Suggest to keep same key word "switch_type" everywhere as it is in sync with METADATA in config json ?

Reply:
I actually changed it to 'chassis type' to match the output of "show platform summary" and "show platform summary --json". 
For 'show platform summary' it makes more sense to use "chassis type" as non chassis switches are also 'switches'. "voq", "non-voq" applies only to 'chassis'.

Reply:
sonic-utilities PR: https://github.com/Azure/sonic-utilities/pull/1922

Reply:
resolved